modules/openapi-generator/src/main/resources/typescript/index.mustache

```diff
@@ -1,41 +1,45 @@
+{{#platforms}}
+{{^deno}}
 import "es6-promise/auto";
```

Comment:
Also curious why we enforce this polyfill instead of delegating that to the end consumer (which might already have a different polyfill in their app!)

It's not a comment about this PR, though - it's actually about the typescript generator already in master :)

Reply:
I remove it and try running the integration test.